common/services/catalogue/api.js

```diff
@@ -49,7 +49,7 @@ export function worksRouteToApiUrl(
   };
 }
 
-export const defaultWorkTypes = ['a', 'b', 'g', 'i', 'k', 'l', 'q'];
+export const defaultWorkTypes = ['a', 'g', 'i', 'k', 'l', 'q'];
```

Comment:
Why are we getting rid of this?

Reply:
it's waiting on the stuff that this PR puts back behind a toggle - we thought we were going to release it, but need to discuss tracking first